sunpy/map/mapbase.py

```diff
@@ -254,8 +254,11 @@ def _text_summary(self):
                                                     self._reference_latitude)),
                                tmf=TIME_FORMAT)
 
+    def __str__(self):
+        return object.__str__(self) + "\n" + self._text_summary() + "\n" + self.data.__str__()
```

Comment:
Why is `__str__` now the old `__repr__`? Why can't it just be the same as `__repr__`? I'm not necessarily against this, I'm just not sure I fully understand the logic here.

Reply:
`__str__()`, unlike `__repr__()`, shouldn't return the object-level representation.

```suggestion
        return f"{self._text_summary()}\n{self.data.__repr__()}"
```
The first part (`object.__str__(self)`) shouldn't be there.  And it should stay `self.data.__repr__()` instead of `self.data.__str__()` because (I feel) we want to preserve the current output when it gets wrapped for `__repr__()`.

Reply:
repr should not change then?

Reply:
I don't understand your question.  To be clear, here's what I'm suggesting:
```python
    def __str__(self):
        return f"{self._text_summary()}\n{self.data.__repr__()}"

    def __repr__(self):
        return f"{object.__repr__(self)}\n{self}"
```